includes/validation/class-amp-validation-error-taxonomy.php

```diff
@@ -1903,8 +1903,15 @@ public static function render_single_url_error_details( $validation_error, $term
 						<div class="detailed">
 							<?php if ( is_string( $value ) ) : ?>
 								<?php echo esc_html( $value ); ?>
-							<?php else : ?>
-								<pre><?php echo esc_html( wp_json_encode( $value, 128 /* JSON_PRETTY_PRINT */ | 64 /* JSON_UNESCAPED_SLASHES */ ) ); ?></pre>
+							<?php elseif ( is_array( $value ) ) : ?>
```

Comment:
This looked to have changed the display of the single error URL details:

<img width="1099" alt="single-error-url" src="https://user-images.githubusercontent.com/4063887/45908336-7e077580-bdd2-11e8-93be-339299cbb64a.png">

Reply:
Addressed with https://github.com/Automattic/amp-wp/pull/1449/commits/5ebc340e40ea9c0ccd744e66a80490fc4f299875:

<img width="1099" alt="single-error-url" src="https://user-images.githubusercontent.com/4063887/45908565-95932e00-bdd3-11e8-98d2-66661a982f1a.png">

Reply:
Thanks for catching that, @kienstra

Reply:
All good, thanks a lot for applying this UI.